src/sentry/interfaces/security.py

```diff
@@ -173,7 +173,10 @@ def to_python(cls, data, **kwargs):
         return super().to_python(data, **kwargs)
 
     def to_string(self, is_public=False, **kwargs):
-        return json.dumps({"csp-report": self.get_api_context()})
+        return orjson.dumps(
+            {"csp-report": self.get_api_context()},
+            option=orjson.OPT_UTC_Z | orjson.OPT_NON_STR_KEYS,
```

Comment:
Why do we have these options here, but not in `stringify` above?

Reply:
Because on `message.py` it gets executed only for int, float and bool, and there is no timestamp or non-string key to serialize. Hence not needed there.

Reply:
I did a quick search for `stringify` and it doesn't seem to be used at all, so maybe we can remove it? Though I might be missing some Python magic.

Reply:
I'll follow up on a different PR to remove this. Thanks for the review!